### ISSUE TITLE
Is there a way to specify source path?

### ISSUE THREAD

**Issue by sammkj** [fr1zle/vscode-elixir#82]
My use-case is I have a monorepo which has the following project structure:

```
\root
  \source
    \elixir
    \javascript
  ...other folders
```

Currently, autocomplete doesn't seem to work with this project structure.

**Comment by fr1zle** [comment]
You are right, this is currently not supported. Should be quite easy though, i will look into this after the weekend.

**Comment by fr1zle** [comment]
We should also try to auto detect the folder including the mix project.

**Comment by thdxr** [comment]
Any update on this? Thanks

**Comment by fr1zle** [comment]
Not yet, sorry. I have this planned for the next release though. Currently trying to figure out how to solve #97

**Comment by fr1zle** [comment]
#97 has been solved and this will be available in the next release. You will be able to specify the setting `elixir.projectPath` in the `.vscode/settings.json` of your project.

This will only allow you to specify one mix project though. The workspaces feature (opening multiple projects in one vscode window) is currently not supporting this.

If we can't find a mix project in the specified folder, there will be a warning message.

The setting for the example in the OP would be (assuming `root` is open in vscode): 

`"elixir.projectPath": "/source/elixir"`
